src/org/usfirst/frc/team2335/robot/commands/AutoDriveTo.java

```diff
@@ -0,0 +1,43 @@
+package org.usfirst.frc.team2335.robot.commands;
+
+import org.usfirst.frc.team2335.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+public class AutoDriveTo extends Command
+{
+    public AutoDriveTo()
+    {
+        requires(Robot.driveTrain);
+        requires(Robot.ultraSound);
+    }
+
+    protected void initialize()
+    {
+    	
+    }
+
+    protected void execute()
+    {
+    	
```

Comment:
Is there supposed to be nothing in the execute method? Wouldn't the command just do nothing?